### ISSUE TITLE
Validation failure for kubernetes.core.helm_repository

### ISSUE THREAD

**Issue by fmunteanu** [ansible/ansible-lint#4171]
##### Summary

While running the `ansible-lint` GitHub workflow, configured as per [documentation](https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/rules/syntax_check.md), an error is produced.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

Github Action, running on `ubuntu-latest`, see [workflow](https://github.com/axivo/k3s-cluster/blob/9e9f3c065d5cdddf4e9e6978ab2fec62bafdfb82/.github/workflows/code.yml) and [requirements](https://github.com/axivo/k3s-cluster/blob/9e9f3c065d5cdddf4e9e6978ab2fec62bafdfb82/collections/requirements.yaml) file.

##### STEPS TO REPRODUCE

See failed [workflow](https://github.com/axivo/k3s-cluster/actions/runs/9117224796/job/25067592790?pr=86):

```
Error: couldn't resolve module/action 'kubernetes.core.helm_repository'. This often indicates a misspelling, missing collection, or incorrect module path.
```

##### Desired Behavior

No errors should be produced, since `kubernetes.core` is listed into [requirements](https://github.com/axivo/k3s-cluster/blob/9e9f3c065d5cdddf4e9e6978ab2fec62bafdfb82/collections/requirements.yaml) file.

##### Actual Behavior

Errors are produced, related to [`kubernetes.core.helm_repository`](https://docs.ansible.com/ansible/latest/collections/kubernetes/core/helm_repository_module.html) module. For example, there no errors displayed for `kubernetes.core.helm` or `kubernetes.core.k8s`, see [role example](https://github.com/axivo/k3s-cluster/blob/feature/ansible-lint/roles/certmanager/tasks/main.yaml). It might be related to the underscore present into `helm_repository`.

Please note there are no linting issues, while running `ansible-lint` on Visual Studio Code.

**Comment by fmunteanu** [comment]
@ssbarnea I'm currently using @v6, which is the latest version as per [documentation](https://github.com/marketplace/actions/run-ansible-lint):

```
jobs:
  validation:
    name: Validation
    runs-on: ubuntu-latest
    steps:
      - name: Checkout Repo
        uses: actions/checkout@v4
      - name: Run Ansible Lint
        uses: ansible/ansible-lint@v6
```
